src/pages/blog/videos/_data.js

```diff
@@ -0,0 +1,92 @@
+export const video_data = [
+    {
+        image: 'https://source.unsplash.com/random/1',
+        video_duration: '1:20',
+        category: ['Multipliers', 'CFD', 'Stocks', 'Forex', 'Category', 'Sixth'],
+        title: 'How to trade multipliers on Deriv',
+        description:
+            'To help you begin, we will look at the basics of cryptocurrency: what it is, how it works, and how to start trading cryptos on Deriv.com.',
+        date: '12 June 2021',
+        id: 1,
```

Comment:
This index is for the `key` child props, correct? we can remove this and probably use other unique props like the `image` object instead?

Reply:
Sure, although this is temporary. Will be replaced with the actual data's UUID in the future.

Reply:
Niceee then LGTM 👍🏼